src/Cms/PageActions.php

```diff
@@ -247,6 +251,19 @@ protected function changeStatusToUnlisted()
         return $page;
     }
 
+    /**
+     * Change the position of the page in its siblings
+     * collection. Siblings will be resorted. If the page
+     * status isn't yet `listed`, it will changed to it.
+     *
+     * @param int|null $position
+     * @return self
+     */
+    public function changeSort(int $position = null)
```

Comment:
As this can be `null`, it should be made explicit for consistency:

```suggestion
    public function changeSort(?int $position = null)
```

Reply:
But... the consistent way would be without the `?` as we don't use it in most places (since it's a more recent feature compared to most of our code base).